includes/discovery/mempools/fiberhome-switch.inc.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * LibreNMS - FiberHome Switch device support - mempools module
+ *
+ * @category   Network_Monitoring
```

Comment:
Please use the copyright header we have in our docs:

https://docs.librenms.org/#Developing/Licensing/#copyright

Reply:
noted and done.